packages/web/src/common/store/player/sagas.ts

```diff
@@ -399,7 +434,51 @@ function* recordListenWorker() {
       if (trackId) yield* put(recordListen(trackId))
       lastSeenPlayCounter = playCounter
     }
-    yield* delay(RECORD_LISTEN_INTERVAL)
+
+    if (position) yield* delay(RECORD_LISTEN_INTERVAL)
```

Comment:
just a check that position can equal 0, so want to double check if that case matters here

Reply:
Can't remember why this was added so I'll remove the check